lib/extras/enc/jxl.cc

```diff
@@ -306,10 +307,9 @@ bool EncodeImageJXL(const JXLCompressParams& params, const PackedPixelFile& ppf,
         const char* type;
         const std::vector<uint8_t>& bytes;
       } boxes[] = {
-          {"Exif", exif_with_offset},
-          {"xml ", ppf.metadata.xmp},
-          {"jumb", ppf.metadata.jumbf},
-          {"xml ", ppf.metadata.iptc},
+          {"Exif", exif_with_offset},   {"xml ", ppf.metadata.xmp},
+          {"jumb", ppf.metadata.jumbf}, {"xml ", ppf.metadata.iptc},
```

Comment:
Not really related to this pull request, but we should probably get rid of `ppf.metadata.iptc` — IPTC is a legacy metadata format that we don't really support; semantically it is a subset of XMP (and Exif, iirc), but syntactically it is different. Applications are supposed to use Exif or XMP instead. Just putting an IPTC blob in an `xml `box is wrong though. We should just remove it from our code.